README.md

```diff
@@ -1,75 +1,64 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
+[](https://img.shields.io/badge/Microverse-blueviolet)
 
 # Project Name
 
-> One paragraph statement about the project.
+> Capstone projects
 
 ![screenshot](./app_screenshot.png)
 
-Additional description about the project and its features.
+This is my microverse module capstone project, a site that i created for a chess tournament.
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- HTML
+- CSS
+- Nodejs
+- NPM
 
 ## Live Demo
 
-[Live Demo Link](https://livedemo.com)
-
-
-## Getting Started
-
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+explanation video:
+https://www.loom.com/share/9be9e71c09824f67ad244e16a35f8eca
 
+## Live Demo
 
-To get a local copy up and running follow these simple example steps.
+The project is deployed using github pages.
 
-### Prerequisites
+[Live Demo Link](https://timowest12.github.io/capstone/)
 
-### Setup
+## Getting Started
 
-### Install
+If you want to use this portfolio as a template, you can do so; follow the instructions below.
 
 ### Usage
 
-### Run tests
-
-### Deployment
-
+Download it directly by ZIP, open the index file in a text editor, and you can change the content to match your own.
+If you would like to add your own projects, you can click on the button that says 'code' on the top of this repository, and clone it to your local machine. Or fork this repo and use it on Github.
+After downloading you can directly open the index.html file in any kind of text editor to make adjustments. Or open in a web browser to view your new site.
```

Comment:
- [ ] This part `getting started` should contain details to the non-technical person on how they should proceed, like go to then do.